services/workflows-service/src/workflow/hook-callback-handler.service.ts

```diff
@@ -363,7 +363,11 @@ export class HookCallbackHandlerService {
     // @ts-expect-error - we don't validate `context` is an object
     this.setNestedProperty(context, attributePath, result);
     // @ts-expect-error - we don't validate `context` is an object
-    context.documents = persistedDocuments;
+    context.documents = [
+      // @ts-expect-error - we don't validate `context` is an object
+      ...context.documents,
+      ...persistedDocuments,
+    ];
```

Comment:
Now both the collection flow and the documents from the KYC process are rendered. Their properties are still not rendered though.